upup/models/config/components/kubelet/_networking_cni/_k8s_1_4/kubelet.cni.1_4.options

```diff
@@ -0,0 +1,8 @@
+Kubelet:
+  # AWS MTU is 9001
+  NetworkPluginMTU: 9001
+
+# TODO: Having to duplicate MasterKubelet & Kubelet feels wrong
```

Comment:
What feels wrong here? Generally vagrant `TODO`'s make me at least want to raise a question :)